### PR TITLE
ci: test ngcc with different target formats

### PR DESCRIPTION
**Pull request by gkalpak**
Jira issue: [FW-1783](https://angular-team.atlassian.net/browse/FW-1783)

##
Blocked on:
- [x] ~~angular/angular#36284~~
- [x] ~~angular/angular#36370~~
- [x] ~~angular/angular#36372~~
- [x] ~~angular/angular#36418~~

### CODE REVIEW

**Comment by gkalpak** [comment]
Actually, this is still wip - not ready for review :grin:

**Comment by gkalpak** [comment]
There seems to be one more failure: `angular-draggable-droppable` not being processed correctly in UMD. Looking into that...

File: ngcc.config.base.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  packages: {
+    // This is not strictly necessary, because `ngcc`'s `CommonJsReflectionHost` (that ends up
+    // handiling the `main` property here) inherits from `Esm5ReflectionHost`, which can handle the
```
Comment (petebacondarwin) [inline ngcc.config.base.js:4]:
```suggestion
    // handling the `main` property here) inherits from `Esm5ReflectionHost`, which can handle the
```

**Comment by gkalpak** [comment]
It turned out that `angular-draggable-droppable` uses Rollup, which generates a UMD format that ngcc does not recognise. See 477328fd1430f7d0225ad61d9df3442afe2e4f0d for details.

@petebacondarwin, ptal at the last commit.

**Comment by gkalpak** [comment]
Thx for reviewing, @petebacondarwin 🤩

**Pull request merged**